MHCollapsibleViewManager.m

```diff
@@ -0,0 +1,228 @@
+//
+//  MHCollapsibleViewManager.m
+//  This is a manager of a uiview and view controller
+//
+//  Created by Lizzy Randall on 5/6/15.
+//  Copyright (c) 2015 Lizzy Randall. All rights reserved.
+//
+
+#import "MHCollapsibleViewManager.h"
+#import "MHFilterLabel.h"
+
+@implementation MHCollapsibleViewManager
+
+//@synthesize tableView;
+//@synthesize viewController;
+@synthesize rowAnimation;
+
+@synthesize data;
+@synthesize cellStyles;
+@synthesize headerTitle;
+
+@synthesize expanded;
+@synthesize numOfRows;
+
+//Initialize with animation, title and tableView
+- (id)initalizeManager:(UITableViewRowAnimation)animation title:(NSString*) title tableView:(UITableView*)tableView
+{
+    self.headerTitle = title;
+    self.rowAnimation = animation;
+    //initialized first and then can be overwritten by controller
+    //tableView is needed to create cells to store types
+    [self setCellProperties:nil header:nil clickedCell:nil normalCell:nil tableView:tableView];
+    return self;
+}
+
+//called after Initializing Manager
+- (void) initializeData:(NSMutableArray*)dataFromController
+{
+    self.data = [[NSMutableArray alloc] init];
+    for(int i=0; i < [dataFromController count] ; i++)
+    {
+        [data addObject:[MHFilterLabel constructLabel:[dataFromController objectAtIndex:i] checked:false]];
+    }
+    self.numOfRows = (NSInteger)[data count];
```

Comment:
This is fine, but in more complex examples where data might have it's length changed by other methods it would be better for numOfRows to be a method instead of a property.
eg
- (NSInteger)numOfRows {
  return self.data.count;
}
methods without parameters can still be used with dot notation so you would still be able to do manager.numOfRows; even if it's a method.